performance-tools/performance-cluster/runner.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+
+set -o pipefail
+
+SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+if [ -f "../../prow/scripts/library.sh" ]; then
+    source "../../prow/scripts/library.sh"
+
+elif [ -f "../test-infra/prow/scripts/library.sh" ]; then
+    source "../test-infra/prow/scripts/library.sh"
+
+else
+	echo "File '/prow/scripts/library.sh' does not exists."
+    exit 1;
+fi
+
+# Set the kubeconfig
+
+if [[ "${KUBERNETES_SERVICE_HOST}" == "" ]]; then
+    shout "KUBERNETES_SERVICE_HOST not set !!!"
+    exit 1
+fi
+
+if [[ "${SLACK_TOKEN}" == "" || "${SLACK_URL}" == "" || "${SLACK_CHANNEL}" == "" ]]; then
+  shout "Slack details not set!! Exiting"
+  exit 1
+fi
+
+if [[ "${GRAFANA_URL}" == "" ]]; then
+  shout "Grafana URL not set!! Exiting"
+  exit 1
+fi
+
+export SRC_DIR="$(mktemp -d -t src.XXXXXX)"
+
+
+# Create Kyma Cluster
+${SCRIPTS_DIR}/cluster.sh --action create --cluster-grade production
+if [[ $? != 0 ]]; then
+shoutFail "Cluster creation failed!!"
+curl -X POST \
+ -H 'Content-type: application/json; charset=utf-8' \
+ --data '{"channel":"'"${SLACK_CHANNEL}"'","text":"Test Run: Failure \n Date: '"${DATE}"' \n Revision: '"${REVISION}"' \n Reason: Cluster Creation Failed"}' \
+ $SLACK_URL/$SLACK_TOKEN
+ exit 1
+fi
+
+export REVISION="$(cd /${SRC_DIR}/kyma-project/kyma && git rev-parse --short HEAD)"
+
+# Get virtualservice
+# Switch to kubeconfig from kyma cluster
+gcloud container clusters get-credentials "${INPUT_CLUSTER_NAME}" --zone="${CLOUDSDK_COMPUTE_ZONE}" --project="${CLOUDSDK_CORE_PROJECT}"
+
+# Get Virtual Service Host Name
+export VIRTUAL_SERVICE_NAME="$(kubectl get virtualservice core-console -n kyma-system -o jsonpath='{.spec.hosts[0]}')"
+
+export CLUSTER_DOMAIN_NAME="${VIRTUAL_SERVICE_NAME#*.}"
+
+if [[ "${CLUSTER_DOMAIN_NAME}" == "" ]]; then
+    shoutFail "Environment CLUSTER_DOMAIN_NAME is empty"
+    exit 1
+fi
+
+shout "Domain name of the kyma cluster is: ${CLUSTER_DOMAIN_NAME}"
+
+# Add the xip.io self-signed certificate to Linux trusted certificates -- Linux (Ubuntu, Debian)
+DIR_SHARED_CERT="/usr/local/share/ca-certificates"
+
+if [ ! -d "${DIR_SHARED_CERT}" ];
+then
```

Comment:
Inconsistent with other if's, `then` should be in the same line, as it is e.g. in line 60 or 85.